package.json

```diff
@@ -17,7 +17,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/heroku/node-js-getting-started"
+    "url": "https://github.com/Manveer695/susi_viberbot"
```

Comment:
The repository linked to heroku should have my repo's address or fossasia's repo address?